addons/block_code/block_code_plugin.gd

```diff
@@ -56,6 +56,7 @@ func _enter_tree():
 
 	# Custom Project->Tools menu items.
 	add_tool_menu_item(tr("Regenerate %s POT file") % "BlockCode", TxUtils.regenerate_pot_file)
+	add_tool_menu_item(tr("Update %s translated files") % "BlockCode", TxUtils.update_pot_files)
```

Comment:
I wonder how feasible it would be to have a CI job that runs a (real, not headless) editor and then executes these...

Reply:
Good questoin. The `TxUtils.update_pot_files` one is independent of the editor, so you could write a headless `SceneTree` script that we could run from CI. I almost started doing it but decided to wait for now since `TxUtils.regenerate_pot_file` is very much dependent on the editor and I had no idea if that could be scripted.

Reply:
One for another day I think.